src/_guides/language/effective-dart/usage.md

```diff
@@ -811,27 +813,27 @@ instead of throwing the same exception object using `throw`.
 `rethrow` preserves the original stack trace of the exception. `throw` on the
 other hand resets the stack trace to the last thrown position.
 
-<div class="bad">
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (rethrow)" replace="/throw e/[!$&!]/g"?>
 {% prettify dart %}
 try {
   somethingRisky();
-} catch(e) {
-  if (!canHandle(e)) throw e;
+} catch (e) {
```

Comment:
The `catch(e)` to `catch (e)` change here (and in the next few examples below) was done by dartfmt.